PubnativeLite/PubnativeLite/AppMonet/AMAppMonetAdView.m

```diff
@@ -22,19 +22,23 @@
 
 #import "AMAppMonetAdView.h"
 #import "AMOConstants.h"
+#import "AMMonetBid.h"
+#import "HyBidLogger.h"
+#import "HyBidAdCache.h"
 
 CGSize const MONET_BANNER_SIZE = {.width = 320.0f, .height = 50.0f};
 CGSize const MONET_MEDIUM_RECT_SIZE = {.width = 300.0f, .height = 250.0f};
 
-@interface AMAppMonetAdView () <HyBidAdViewDelegate>
+@interface AMAppMonetAdView () <HyBidAdViewDelegate, HyBidAdRequestDelegate>
 @property (nonatomic) CGSize size;
 @end
 
 @implementation AMAppMonetAdView
 
-- (id)initWithAdUnitId:(NSString *)adUnitId size:(CGSize)size {
-    HyBidAdSize *newSize = [[[HyBidAdSize alloc] init] convertSizeToHyBidAdSize:size];
-    self = [self initWithSize:newSize];
+- (id)initWithAdUnitId:(NSString *)adUnitId size:(HyBidAdSize *)size {
```

Comment:
Here we shouldn't replace the interface with our class. We should do the mapping internally.

Reply:
Changing this from CGSize to HyBidAdSize will break existing publisher's integration

Reply:
@eros902002 true! I changed it to test something and forgot return back. One moment :)

Reply:
@can-soykarafakili yes! I forgot to change it back 🌚

Reply:
@orkhanalizada2020 I think you can copy & paste the size mapping code that I've wrote in `AMCustomEventBanner` under AppMonet Adapters/MoPub Adapters 👍

Reply:
@can-soykarafakili I've added it in my latest commit. Please check it

Reply:
Looks okay @orkhanalizada2020  👍